spec/lib/karafka/serialization/json/serializer_spec.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+RSpec.describe Karafka::Serialization::Json::Serializer do
+  subject(:serializer) { described_class.new }
+
+  describe '.call' do
+    context 'when content is a string' do
+      let(:content) { rand.to_s }
+
+      it 'expect not to do anything' do
+        expect(serializer.call(content)).to eq content
+      end
+    end
+
+    context 'when content can be serialied with #to_json' do
```

Comment:
`serialied` -> `serialized` here and below on line 23

Reply:
Thanks, fixed.